### DISCUSSION TITLE
Publish sensor data to MQTT

### DISCUSSION THREAD

**Discussion by phrozen77** [custom-components/nordpool#458] [Ideas]
As the topic says, I’d like to publish either any and all or especially the today_raw and tomorrow_raw sensor values to a to be specified MQTT topic, ideally under the default „homeassistant“ one.

My reasoning behind this is for it to be used in evcc as price forecast data on when to charge my EV.

Has anyone already done this yet? I’m struggling to provide the correct syntax to a „mqtt.publish“ automation so I’d appreciate any help I can get. Ideally, this would work directly from within the Nordpool integration itself..

**Comment by Hellowlol** [comment]
It’s out of scope of the integration and I will never add support for it. It would be easier to assist you if you can provide some examples for what you have already tried.
